.travis.yml

```diff
@@ -98,7 +103,10 @@ script:
       rustup component add rustfmt && cargo fmt --all -- --check;
       scripts/generate-bindings.sh && git diff --exit-code HEAD;
     fi
-  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then scripts/lint-grpc-sys.sh && git diff-index --quiet HEAD; fi
+  # clang-tidy-7 is not available on aarch64, so just skip this step
+  - if [[ $TRAVIS_OS_NAME == "linux" ]] && [[ `uname -p` != "aarch64" ]];
+      then scripts/lint-grpc-sys.sh && git diff-index --quiet HEAD; 
```

Comment:
skipping `git diff` may cause problems, you can separate these two steps

Reply:
fixed

Reply:
I'm sorry I haven't seen the `git diff` operation above. So pls roll back to your original commit